Example/S2MToolbox/S2MTextLoadingView.m

```diff
@@ -0,0 +1,81 @@
+//
+//  S2MTextLoadingView.m
+//  S2MToolbox
+//
+//  Created by François Benaiteau on 12/12/14.
+//  Copyright (c) 2014 Sinnerschrader Mobile. All rights reserved.
+//
+
+#import "S2MTextLoadingView.h"
+#import <S2MToolbox/UIView+S2MAutolayout.h>
+
+@implementation S2MTextLoadingView
+
+- (instancetype)init
+{
+    self = [super initWithFrame:CGRectMake(0, 0, 250, 30)];
+    if (self) {
+        self.label = [[UILabel alloc] init];
+        self.label.text = @"Loading...";
+        [self addSubview:self.label];
+        [self.label sizeToFit];
+        self.label.textColor = [UIColor whiteColor];
+        self.label.alpha = 0;
+    }
+    return self;
+}
+
+- (void)layoutSubviews
+{
+    [super layoutSubviews];
+    self.label.center = CGPointMake(CGRectGetMidX(self.bounds), CGRectGetMidY(self.bounds));
+}
+
+#pragma mark - <S2MControlLoadingView>
+
+- (void)startAnimating
+{
+    [UIView animateWithDuration:0.3 animations:^{
+        self.label.alpha = 1;
+    }];
+
+    self.timer = [NSTimer scheduledTimerWithTimeInterval:1 target:self selector:@selector(timerFires:) userInfo:nil repeats:YES];
```

Comment:
@joernito this class is not part of the toolbox just an example and has no intention to be reused. so I would leave it as it is